controllers/networkchaos/podnetworkmanager/podnetworkmanager.go

```diff
@@ -83,7 +93,15 @@ func (m *PodNetworkManager) Commit(ctx context.Context) error {
 					pod := v1.Pod{}
 					err = m.Client.Get(ctx, key, &pod)
 					if err != nil {
-						m.Log.Error(err, "error while finding pod", "key", key)
+						m.Log.Info("pod not found", "key", key, "error", err.Error())
+						err = ErrPodNotFound
+						return err
+					}
+
+					if pod.Status.Phase != v1.PodRunning {
```

Comment:
These pod check code is nice-to-have thing, since we does the retry [below](https://github.com/chaos-mesh/chaos-mesh/pull/946/files#diff-6688ff79c2c81a4041a2754548d442c8R141), but I think networkchaos should not happens on invalid pods(not found, not running).

Anyway, feel free to ask me delete these code if we think it's useless.